README.md

```diff
@@ -1,53 +1,53 @@
-# Holochain
-
-[![Project](https://img.shields.io/badge/project-holochain-blue.svg?style=flat-square)](http://holochain.org/)
-[![Forum](https://img.shields.io/badge/chat-forum%2eholochain%2enet-blue.svg?style=flat-square)](https://forum.holochain.org)
-[![Chat](https://img.shields.io/badge/chat-chat%2eholochain%2enet-blue.svg?style=flat-square)](https://chat.holochain.org)
-
+[![Project](https://img.shields.io/badge/Project-Holochain-blue.svg?style=flat-square)](http://holochain.org/)
+[![Forum](https://img.shields.io/badge/Forum-forum%2eholochain%2enet-blue.svg?style=flat-square)](https://forum.holochain.org)
+[![License: CAL 1.0](https://img.shields.io/badge/License-CAL%201.0-blue.svg)](https://github.com/holochain/cryptographic-autonomy-license)
+![Test](https://github.com/holochain/holochain-client-js/actions/workflows/test.yml/badge.svg?branch=main)
 [![Twitter Follow](https://img.shields.io/twitter/follow/holochain.svg?style=social&label=Follow)](https://twitter.com/holochain)
-License: [![License: CAL 1.0](https://img.shields.io/badge/License-CAL%201.0-blue.svg)](https://github.com/holochain/cryptographic-autonomy-license)
 
-This repository contains the core Holochain libraries and binaries.
+# Holochain
 
-This is the most recent and well maintained version of Holochain with a refactored state model (you may see references to it as Holochain RSM).
+Holochain is an open-source framework to develop peer-to-peer applications with high levels of security, reliability, and performance.
+This repository contains the Holochain core libraries.
 
 ## Code Status
 
-This code is in alpha. It is not for production use. The code is guaranteed NOT secure.
+This code is in alpha. It is not for production use. We will be frequently and heavily restructuring code APIs and data chains until Beta.
 
-We will be frequently and heavily restructuring code APIs and data chains until Beta.
+**We are supporting Linux and macOS at this time**. You may or may not be able to build and run Holochain on Windows.
```

Comment:
we're not running any tests for macOS so we are not guaranteeing functionality beyond being able to obtain an executable.

Reply:
@steveeJ Okay - on a side note: You could try that again with nextest to see if it's any better now.